src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/overloadmethodsdeclarationorder/InputOverloadMethodsDeclarationOrder2.java

```diff
@@ -0,0 +1,99 @@
+/*
+OverloadMethodsDeclarationOrder
+modifierGroups = .*
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.overloadmethodsdeclarationorder;
+
+class InputOverloadMethodsDeclarationOrder2
+{
+    public void overloadMethod(int i) { }
+
+    public void overloadMethod(String s) { }
+
+    public void overloadMethod(boolean b) { }
+
+    public void fooMethod() { }
+
+    //violation: because overloads never split
+    public void overloadMethod(String s, Boolean b, int i) // violation
```

Comment:
Is the full violation message required in the test file as well? I can't find any examples of the violation message being multiple lines, and I can't get it to work no matter what I try. It seems like the check is only validating a subset of the message. For example, the below passes
```java
    //violation below 'All overloaded methods should be placed next to each other.'
    public void overloadMethod(String s, Boolean b, int i) {

    }
```

The full violation message is quite unwieldy
```
All overloaded methods should be placed next to each other. Placing non-overloaded methods in between overloaded methods with the same type is a violation. Previous overloaded method located at line '15'.
```

Reply:
You should be able to do something like:
```
    //violation below 'All overloaded methods should be placed .* Previous overloaded method located at line '15'.'


```

Reply:
@nrmancuso I don't think this syntax supports multiple lines, see https://github.com/checkstyle/checkstyle/blob/f7ce9ad82249706813e2ec3a454205257566d2a2/src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java#L88 

It is running the regex check one line at a time, so the comment can spill over onto multiple lines. what do you suggest?

Reply:
I just wanted to make sure cutting out a chunk of the message with `.*` to make it fit on a single line would be ok, thanks for clarifying. Does the line number at the end of the error need to be included in the BDD comment? IMO we don't need it because it's being included in the test code, but if you would prefer to have it in the comment as well i'll make sure to include it in the BDD comment also

Reply:
> IMO we don't need it because it's being included in the test code

Agreed.